runtime/kusama/src/governance/tracks.rs

```diff
@@ -22,14 +22,20 @@ const fn percent(x: i32) -> sp_arithmetic::FixedI64 {
 	sp_arithmetic::FixedI64::from_rational(x as u128, 100)
 }
 use pallet_referenda::Curve;
-const APP_ROOT: Curve = Curve::make_reciprocal(4, 28, percent(80), percent(50), percent(100));
-const SUP_ROOT: Curve = Curve::make_linear(28, 28, percent(0), percent(50));
-const APP_STAKING_ADMIN: Curve = Curve::make_linear(17, 28, percent(50), percent(100));
+const APP_ROOT: Curve =
+	Curve::make_reciprocal(4, 28, percent(80), percent(50), percent(100));
```

Comment:
The format changes make this a bit difficult to review. I will try `bot fmt` maybe it has write access.

Reply:
The bot has no access. Could you please format the code with the same version as in the CI, `cargo +nightly-2023-05-16 fmt`?

Reply:
Hi, sorry for the formatting! I've reverted the formatting back to what it used to be to make comparison easier.